content/extraProblems.json

```diff
@@ -835,6 +835,71 @@
         "usacoId": "1214"
       }
     },
+    {
+      "uniqueId": "usaco-1228",
+      "name": "Counting Liars",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1228",
+      "source": "Bronze",
+      "difficulty": "Easy",
+      "isStarred": false,
+      "tags": ["Complete Search"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1228"
+      }
+    },
+    {
+      "uniqueId": "usaco-1229",
+      "name": "Alchemy",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1229",
+      "source": "Bronze",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Simulation", "Complete Search", "Recursion"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1229"
+      }
+    },
+    {
+      "uniqueId": "usaco-1230",
+      "name": "Visits",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1230",
+      "source": "Silver",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["SCC"],
```

Comment:
^ not intended ...

Reply:
should i put this in the functional graphs module?

Reply:
k